### ISSUE TITLE
When I enable backlog I cant use recurrence for any task in project with backlog enabled

### ISSUE THREAD

**Issue by GusevVictor** [vistree/kanboard-backlog#19]
### Configuration

    Application version: 1.2.9
    PHP version: 7.2.19-0ubuntu0.18.04.1
    PHP SAPI: apache2handler
    OS version: Linux 4.15.0-51-generic
    Database driver: postgres
    Database version: 10.8 (Ubuntu 10.8-0ubuntu0.18.04.1)
    Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:60.0) Gecko/20100101 Firefox/60.0
    Backlog 1.0.3

There is a big problem with recurrence for any task in project where backlog is enabled.

### Step to repeat:

1. Enable backlog for a project
2. Create a task in this project (in any coloum)
3. Add recurrence for that task (with any trigger for recurrence)
4. Activate a recurrence trigger: _Move task/close task/ and so on_
5. A new task will be create in backlog but will not show in board mode view (but count will be +1) of a project but will show in list - please see an image

Please fix it. Thank a lot for you job

![изображение](https://user-images.githubusercontent.com/7081357/59690711-85856f00-91ea-11e9-8ba3-c2152d4d80cb.png)

**Comment by creecros** [comment]
ugh...it's the little things.

Task Recurrence creates a new task of the one "Moved/Closed/and so on", in the first column, first swimlane. 

That therein lies the problem. Backlog board has automatically hidden the first swimlane from your view.

@vistree 

It could be a while before i get to this. My thoughts would be:
- in the plugin.php, initialize:
  - if backlog is enabled
  - search for tasks in the hidden swimlane
  - move them to appropriate swimlane

Been a long time since I looked at this plugin, so that's the best I can tell you.

**Comment by creecros** [comment]
Well, I was close. 

@GusevVictor 

Test and report back. 

@vistree 

you can update the release. ;)

**Comment by GusevVictor** [comment]
> 
> 
> Well, I was close.
> 
> @GusevVictor
> 
> Test and report back.
> 
> @vistree
> 
> you can update the release. ;)

There is a problem again... But there is little difference behaviour

![изображение](https://user-images.githubusercontent.com/7081357/59705558-4dd8f000-9207-11e9-96ed-eb06a774f868.png)

**Comment by creecros** [comment]
Not sure I understand, what is the trigger?

**Comment by creecros** [comment]
Not sure I understand, what is the trigger?

**Comment by GusevVictor** [comment]
> 
> 
> Not sure I understand, what is the trigger?

Trigger create a new task in first column when move task from first column

![изображение](https://user-images.githubusercontent.com/7081357/59710699-26d3eb80-9212-11e9-9d26-b0ac371bba50.png)

**Comment by creecros** [comment]
ya, i got it.

works on Task Close
works on move to Last Column

does not work on move from first column.

It's not triggering: `TaskModel::EVENT_MOVE_COLUMN => 'onMove',`
When moving from first column.

Currently no idea why.

**Comment by GusevVictor** [comment]
> 
> 
> ya, i got it.
> 
> works on Task Close
> works on move to Last Column
> 
> does not work on move from first column.
> 
> It's not triggering: `TaskModel::EVENT_MOVE_COLUMN => 'onMove',`
> When moving from first column.
> 
> Currently no idea why.

No. Doesn't show a task when I move it to Last Column. I have test 'move to Last Column' trigger

![изображение](https://user-images.githubusercontent.com/7081357/59711889-b24e7c00-9214-11e9-9eed-a19579870f71.png)

**Comment by GusevVictor** [comment]
But I can see it in the list view!

![изображение](https://user-images.githubusercontent.com/7081357/59712259-8a134d00-9215-11e9-9c20-aa56eab89f5f.png)

**Comment by GusevVictor** [comment]
> 
> 
> But I can see it in the list view!
> 
> ![изображение](https://user-images.githubusercontent.com/7081357/59712259-8a134d00-9215-11e9-9c20-aa56eab89f5f.png)

I have use Customize plugin for change theme and so on.

**Comment by creecros** [comment]
Hmmm... I tested and it worked.

**Comment by GusevVictor** [comment]
> 
> 
> Hmmm... I tested and it worked.

My plugins is: 
![изображение](https://user-images.githubusercontent.com/7081357/59712958-fa6e9e00-9216-11e9-9d05-7ccd7c7aaf13.png)

**Comment by creecros** [comment]
ya, it works, when going to last column. 

Tested again, with Customizer loaded. 

I did have to "refresh" my browser though, but being in board mode, it should auto refresh based on your settings.

**Comment by GusevVictor** [comment]
> 
> 
> Hmmm... I tested and it worked.

I have tested it with "close task" trigger no luck...

![изображение](https://user-images.githubusercontent.com/7081357/59713254-a0baa380-9217-11e9-973a-d27ceb649cbe.png)

**Comment by creecros** [comment]
Try restarting your server, sounds like you need to refresh your code.

**Comment by GusevVictor** [comment]
> 
> 
> ya, it works, when going to last column.
> 
> Tested again, with Customizer loaded.
> 
> I did have to "refresh" my browser though, but being in board mode, it should auto refresh based on your settings.

I have full refresh page (no luck)
I have reboot server no luck...

![изображение](https://user-images.githubusercontent.com/7081357/59713482-3b1ae700-9218-11e9-9fc3-0fbe450764f8.png)

**Comment by creecros** [comment]
show me a screenshot of your plugin.php file for backlog

**Comment by creecros** [comment]
prefer to take this offline:
https://nachostudio.ml

**Comment by GusevVictor** [comment]
> 
> 
> show me a screenshot of your plugin.php file for backlog

root@project:~# md5sum /var/www/html/plugins/Backlog/Plugin.php 
41ac37f7031ccf8f6ca35e056c1aba72  /var/www/html/plugins/Backlog/Plugin.php

I have attached Plugin.php file. Change php to txt for github restriction policy
[Plugin.txt](https://github.com/vistree/kanboard-backlog/files/3303098/Plugin.txt)

**Comment by vistree** [comment]
I think this problem is still open, right? So, it does not make sense to create a new release right now??!

**Comment by GusevVictor** [comment]
> 
> 
> I think this problem is still open, right? So, it does not make sense to create a new release right now??!

Yep. I don't have free time to test it. But part of problem still here... And in my current production version this fix doesn't work... :(

**Comment by phleva** [comment]
Hi,
I am having the same problem as described above (recurrence with new task created upon closing). Task appears in list view but not in table view. Any idea regarding resolution? In the mean time I either need to remove backlog or recurrence? Thank you for your help.
